CHANGELOG.md

```diff
@@ -8,11 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
```

Comment:
I don't think it made sense to have refactoring change in same release with addition of the said feature :slightly_smiling_face: